chirho/robust/internals/linearize.py

```diff
@@ -152,16 +152,35 @@ def linearize(
     )
 
     @functools.wraps(score_fn)
-    def _fn(point: Point[T], *args: P.args, **kwargs: P.kwargs) -> ParamDict:
+    def _fn(
+        points: Point[T],
+        pointwise_influence: bool = True,
+        *args: P.args,
+        **kwargs: P.kwargs,
+    ) -> ParamDict:
         with torch.no_grad():
             data: Point[T] = func_predictive(predictive_params, *args, **kwargs)
-            data = {k: data[k] for k in point.keys()}
+            data = {k: data[k] for k in points.keys()}
         fvp = make_empirical_fisher_vp(
             func_log_prob, log_prob_params, data, *args, **kwargs
         )
-
         pinned_fvp = reset_rng_state(pyro.util.get_rng_state())(fvp)
-        point_score: ParamDict = score_fn(log_prob_params, point, *args, **kwargs)
-        return cg_solver(pinned_fvp, point_score)
+        batched_func_log_prob = torch.vmap(
```

Comment:
Oops I forgot to remove the line `score_fn = torch.func.grad(func_log_prob)`, thanks for the catch! I don't think we want to `vmap` over `score_fn` because once #463 is addressed, `func_log_prob` will return the log probability of multiple datapoints at once. So we would need to use `jacrev` or `jacfwd` since the output of `func_log_prob` is a vector. @eb8680